src/ape_geth/providers.py

```diff
@@ -233,8 +233,18 @@ def is_likely_poa() -> bool:
                 or len(block.get("extraData", "")) > MAX_EXTRADATA_LENGTH
             )
 
+        # Check for chain errors, including syncing
+        try:
+            chain_id = self._web3.eth.chain_id
+        except ValueError as err:
+            raise ProviderError(
+                err.args[0].get("message")
+                if (hasattr(err, "args") and err.args)
```

Comment:
maybe add a check to make sure `err.args[0]` is also a `dict`? It can be anything, it is just the arguments to the exception

Reply:
Updated https://github.com/ApeWorX/ape/pull/812/commits/e3e36ebb00f4c43087cd7b5233d3cb0eb4ad7d8b